gratipay/utils/team_history.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+from decimal import Decimal as D
+
+from aspen import Response
+
+
+def get_end_of_year_totals(db, team, year):
+
+    received = db.one("""
```

Comment:
@whit537 I agree for our current work it may not be a must. Generally, here listed some [good points](http://softwareengineering.stackexchange.com/questions/68692/database-abstraction-is-it-being-overdone/68731) about using abstraction.